### DISCUSSION TITLE
Question: Dependency to the Microsoft Visual C++ Redistributables

### DISCUSSION THREAD

**Discussion by TomGruenewald** [icsharpcode/SharpZipLib#610] [Q&A]
Hello.

Question: Is any Microsoft Visual C++ Redistributable required to run the SharpZibLib library?

Thank you for you answers.

Tom

**Comment by piksel** [comment]
The library itself is a CLR (.NET) library and hence does not rely on any C++ runtime, but the .NET runtime you are using might itself need the redistributable.

**Reply by TomGruenewald** [reply]
Thank you.
